aries_cloudagent/resolver/default/tests/test_indy.py

```diff
@@ -40,20 +40,23 @@ def profile(ledger):
     yield profile
 
 
-def test_supported_methods(resolver: IndyDIDResolver):
+@pytest.mark.asyncio
+async def test_supported_methods(profile, resolver: IndyDIDResolver):
     """Test the supported_methods."""
     assert resolver.supported_methods == ["sov"]
-    assert resolver.supports("sov")
+    assert await resolver.supports(
+        profile, "did:sov:U292ZXJlaWdudHksIGxveWFsdHksIGFuZCBzb2xpdHVkZS4="
```

Comment:
If we tightened up the `supports` method for the indy resolver, this DID would technically fail since it is not Base58 in the method specific ID.

Reply:
let's pass that bridge when we come to it.

Reply:
I updated to base58